xpass/xpass.cc

```diff
@@ -371,7 +482,9 @@ void XPassAgent::send_credit() {
 
 void XPassAgent::send_credit_stop() {
   send(construct_credit_stop(), 0);
-  credit_recv_state_ = XPASS_RECV_CLOSED;
+  // set on timer
+  retransmit_timer_.resched(credit_ignore_timeout_);    
```

Comment:
Let's reduce this to 2 RTT if RTT(rtt_) is valid. If RTT(rtt_) is invalid, we can set the default value of up to 1ms.